### ISSUE TITLE
Device selection should be automatic

### ISSUE THREAD

**Issue by merlijn-sebrechts** [Hankamotz/Fmessenger#10]
Doesn't Ubuntu phone provide a way to detect what kind of device/screen you're on, and change the setting accordingly?

**Comment by Hankamotz** [comment]
Yes there is, its called adaptative page layout. It would be perfect if Fmessenger were responsive to different devices, but I haven't  enough coding skill to do that. Currently the Fmessenger is a webapp of messenger (https://www.messenger.com/) and runs different scripts to change the size/positions of elements in the web page according to device selection.
